internal/pkg/agent/install/install.go

```diff
@@ -78,16 +78,17 @@ func Install(cfgFile string) error {
 						errors.M("destination", paths.ShellWrapperPath))
 				}
 			}
-			err = os.Symlink("/Library/Elastic/Agent/elastic-agent", paths.ShellWrapperPath)
+			err = os.Symlink(filepath.Join(topPath, paths.BinaryName), paths.ShellWrapperPath)
 			if err != nil {
 				return errors.New(
 					err,
 					fmt.Sprintf("failed to create elastic-agent symlink (%s)", paths.ShellWrapperPath),
 					errors.M("destination", paths.ShellWrapperPath))
 			}
 		} else {
-			//nolint: gosec // this is intended to be an executable shell script, not changing the permissions for the linter
-			err = os.WriteFile(paths.ShellWrapperPath, []byte(paths.ShellWrapper), 0755)
+			//nolint: govet,staticcheck // the first argument to fmt.Sprintf is a constant containing one formatting directive.
```

Comment:
I do not know how to get rid of the linter errors for this line.  I've tried removing `govet` but then I get a `govet` linter error.